keyboard/src/androidLibMain/kotlin/KeyboardManager.kt

```diff
@@ -22,7 +22,7 @@ Copyright 2019 Splendo Consulting B.V. The Netherlands
 
 */
 
-actual typealias KeyboardView = View
+actual typealias TextInputView = View
```

Comment:
Confused with Text Field, like you can input data into this View

Reply:
I mean, any view that can show a keyboard can probably get TextInput. I think its a fairly clear name. What would you suggest?

Reply:
`KeyboardHostingView` or something like this